lib/PuppeteerSharp/Frame.cs

```diff
@@ -1,647 +1,662 @@
-﻿using System;
```

Comment:
This changelog is too big, can we fix it?

Reply:
It's too big because your line endings were inconsistent, I correct them, not sure what you expect me to do, make it wrong again ?

Reply:
@Meir017 are you getting line ending warnings as well?

We need diffs as small as possible if we need to change line endings we need to do that in a separate PR.

Reply:
The files in question contain a mixture of `LF` and `CRLF`
I don't know what to say, I'm not prepared to revert these files and do all the work over again ?

Reply:
If the entire file is changed we don’t know what to review. We need PRs as small as possible.

If the IDE shows me a warning like that, I create a PR only with that change or ignore it, so we don’t mess with the file history.

Let’s see what @Meir017 is doing with that. I’m on Mac.

Reply:
So, for you to know. We try to make PRs as small as possible and focus on one thing.
We also tend to create an issue first, even us. So we can have a discussion before implementing that change.

So if we think we need to use constants we discuss it, if we see mixed line endings, we discuss it.

So the developer doesn't waste calories implementing something that wasn't expected.

Reply:
Github isn't my 'native' situation, I'm used to diffing that ignores line ending differences.